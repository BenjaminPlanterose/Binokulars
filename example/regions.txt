chr5:372378-374378 # AHRR
chr6:11043877-11045877 # ELOVL2
chr1:31957245-31959245 # Gran_Mon_specific
chr6:291329-293329 # DUSP22