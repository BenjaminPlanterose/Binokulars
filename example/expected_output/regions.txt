chr5:372378-374378 # AHRR
chr6:11043877-11045877 # ELOVL2
chr6:291329-293329 # DUSP22
chr12:739000-741000 # NINJ2
chr1:205818132-205820132 # meQTL2
chr9:124988248-124990248 # meQTL3
chr12:96583672-96585672 # Bcellspecific
